app/index.js

```diff
@@ -42,12 +42,37 @@ module.exports = Base.extend({
         return !self.hasPackageJson;
       }
     }, {
-      name: 'wabRoot',
-      message: 'Web AppBuilder install root:',
-      'default': path.join(homedir, 'arcgis-web-appbuilder-1.3'),
+      type: 'list',
+      choices: [
+        {
+          value: 'is2d',
+          name: '2D'
+        },
+        {
+          value: 'is3d',
+          name: '3D'
+        }
+      ],
+      name: 'widgetsType',
+      message: 'Type of widget(s) to be generated:',
+      when: function(currentAnswers) {
+        return !currentAnswers.abort;
+      }
+    }, {  
       when: function(currentAnswers) {
         return !currentAnswers.abort;
       },
+      name: 'wabRoot',
+      message: 'Web AppBuilder install root:',
+      'default': function(currentAnswers) {
+        var wabDir;
+        if (currentAnswers.widgetsType === 'is2d') {
+          wabDir = path.join(homedir,  'arcgis-web-appbuilder-1.3' );
+        } else if (currentAnswers.widgetsType === 'is3d') {
+          wabDir = path.join(homedir,  'arcgis-web-appbuilder-2.0beta\\WebAppBuilderForArcGIS' );
```

Comment:
Default path for 3d should be `path.join(homedir,  'WebAppBuilderForArcGIS' );` Also, let's use consistent conditional logic that you use below. So it should always be "if 3d then y else x" instead of "if 2d then x else if 3d then y." In other words, let's always default to 2d values.